.gitlab-ci.yml

```diff
@@ -22,65 +28,55 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
 
-test:unit:
-  stage: test
-  before_script:
-    - apt-get -qq update && apt-get install -yq $(cat deb-requirements.txt)
-    # start the dbus service
-    - service dbus start
-
-build:make:
+build:docker:
+  tags:
+    - mender-qa-worker-generic-light
```

Comment:
I wanted to avoid pushing this to the gitlab registry since the images are quite large and they won't be cleaned up. By only saving it locally, the artifacts eventually expire.